docs/source/ui/pages/reports.md

```diff
@@ -0,0 +1,175 @@
+## Reports
+
+### Overview
+
+Use Aim reports to organize runs, embed visualizations, describe your findings, and share updates with collaborators. 
+
+The following image shows an example of a report created from metrics that were logged to aim over the course of a training. 
+<img alt="View Mode" style="border-radius: 8px; border: 1px solid #E8F1FC" src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/reports/view-mode.png">
+
+
+### Single report page
+In single report page you can find 2 modes: `view` and `edit`. 
+
+
+In `edit` mode you can create the report that you wish for and in view mode see the final result of it, the example shared above is of the `view` mode.
```

Comment:
Create the report in  `edit` mode and view the final result in the `view` mode. The example above is on the `view` mode.